src/contracts/protocol/tokenization/RWAAToken.sol

```diff
@@ -0,0 +1,105 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.10;
+
+import {AccessControl} from 'src/contracts/dependencies/openzeppelin/contracts/AccessControl.sol';
+import {IncentivizedERC20} from 'src/contracts/protocol/tokenization/base/IncentivizedERC20.sol';
+import {SafeCast} from 'src/contracts/dependencies/openzeppelin/contracts/SafeCast.sol';
+import {IERC20} from 'src/contracts/dependencies/openzeppelin/contracts/IERC20.sol';
+import {Errors} from 'src/contracts/protocol/libraries/helpers/Errors.sol';
+import {AToken} from 'src/contracts/protocol/tokenization/AToken.sol';
+import {IRWAAToken} from 'src/contracts/interfaces/IRWAAToken.sol';
+import {IPool} from 'src/contracts/interfaces/IPool.sol';
+
+abstract contract RWAAToken is AToken, IRWAAToken {
+  using SafeCast for uint256;
+
+  bytes32 public constant ATOKEN_TRANSFER_ROLE = keccak256('ATOKEN_TRANSFER_ROLE');
```

Comment:
add docs here and at `contract` level

Reply:
[2745aab](https://github.com/aave/aave-v3-horizon/pull/1/commits/2745aab9a8e6bb93fa9d1bd72fbd4c01b8bda2b8)